documentation/Sphinx/user_guide/setting-up/setting-up.rst

```diff
@@ -114,11 +114,23 @@ Environment Variables
 
    ``VERNIER_IO_MODE``
 
-     Determines the output mode to use. Currently only supports being set to 
-     **multi** but single-file-output may be added in the future.
+     Determines the output mode to use. Currently supported values are:
+
+     * **multi**: Every task writes to its own output file
+       
+     * **single**: All tasks write to a single global output file
+
+     If the environment variable is not set, **multi** mode is used.
```

Comment:
We may want to change the default at some point as the single file output is likely to be the preferred method for most.

Reply:
Collated?